test/fibonacci.js

```diff
@@ -10,7 +10,23 @@ describe.only('fibonacci()', function(){
   it('should return an array of length 50, whos 9th value is 34', function(){
```

Comment:
You test doesn't line up with this description. It looks like you are just testing to see if your function returns an array.